app/products/playbooks/database/models/playbook_run.ts

```diff
@@ -0,0 +1,144 @@
+import type UserModel from '@typings/database/models/servers/user';
+
+const {
+    CHANNEL,
+    POST,
+    TEAM,
+    USER,
+} = MM_TABLES.SERVER;
+
+const {PLAYBOOK_RUN, PLAYBOOK_CHECKLIST} = PLAYBOOK_TABLES;
+
+/**
+ * The PlaybookRun model represents a playbook run in the Mattermost app.
+ */
+export default class PlaybookRunModel extends Model implements PlaybookRunModelInterface {
+    /** table (name) : PlaybookRun */
+    static table = PLAYBOOK_RUN;
+
+    /** associations : Describes every relationship to this table. */
+    static associations: Associations = {
+
+        /** A PLAYBOOK_RUN could have been created by a POST (relationship is 1:1) */
+        [POST]: {type: 'belongs_to', key: 'post_id'},
+
+        /** A CHANNEL can be associated to one PLAYBOOK_RUN (relationship is 1:1) */
+        [CHANNEL]: {type: 'belongs_to', key: 'channel_id'},
+
+        /** A TEAM can be associated to PLAYBOOK_RUN (relationship is 1:1) */
+        [TEAM]: {type: 'belongs_to', key: 'team_id'},
+
+        /** A USER that commands the PLAYBOOK_RUN (relationship is 1:1) */
+        [USER]: {type: 'belongs_to', key: 'owner_user_id'},
+
+        [PLAYBOOK_CHECKLIST]: {type: 'has_many', foreignKey: 'run_id'},
+    };
+
+    /** playbook_id : The id of the playbook this run belongs to */
+    @field('playbook_id') playbookId!: string;
+
+    /** name : Name of the playbook run */
+    @field('name') name!: string;
+
+    /** description : Description of the playbook run */
+    @field('description') description!: string;
+
+    /** is_active : Whether the run is still active */
+    @field('is_active') isActive!: boolean;
+
+    /** owner_user_id : Foreign key to the user commanding the run */
+    @field('owner_user_id') ownerUserId!: string;
+
+    /** team_id : Foreign key to the team this run belongs to */
+    @field('team_id') teamId!: string;
+
+    /** channel_id : Foreign key to the channel this run belongs to */
+    @field('channel_id') channelId!: string;
+
+    /** post_id : ID of the post that created the run (nullable) */
+    @field('post_id') postId!: string | null;
+
+    /** create_at : Timestamp when the run was created */
+    @field('create_at') createAt!: number;
+
+    /** end_at: Timestamp when the run ended (0 if not finished) */
+    @field('end_at') endAt!: number;
+
+    /** active_stage : Zero-based index of the currently active stage */
+    @field('active_stage') activeStage!: number;
+
+    /** active_stage_title : Name of the current active stage */
+    @field('active_stage_title') activeStageTitle!: string;
+
+    /** participant_ids : An array of user IDs that participate in the run */
+    @json('participant_ids', safeParseJSON) participantIds!: string[];
+
+    /** summary : Summary of the playbook run */
+    @field('summary') summary!: string;
+
+    /** current_status : The current status of the playbook run */
+    @field('current_status') currentStatus!: PlaybookRunStatusType;
```

Comment:
there are only 2 possible status options: in progress or ended. Do we need to store this info? doesn't it make more sense to infer it from the data like end_date?